sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/ManagementChannel.java

```diff
@@ -289,7 +297,8 @@ private Message createDispositionMessage(UUID[] lockTokens, DispositionStatus di
             }
 
             return Flux.fromIterable(messageSerializer.deserializeList(responseMessage, Instant.class));
```

Comment:
You already have access to the list here, why not return the first item in the list as a mono rather than wrapping it in a flux from iterable and only choosing one item from it?